ParisBNF/abb/BNFabb28.xml

```diff
@@ -0,0 +1,312 @@
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+               Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </p>
+               </licence>
+            </availability>
+
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                 <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b10091322n">BnF Éthiopien d'Abbadie 28</idno>
+                    </msIdentifier>
+               <msContents>
+                  <summary>
+                  </summary>
+
+                  <msItem xml:id="ms_i1">
+                    <locus from="2ra" to="86vb"/>
+                     <title type="complete" ref="LIT6608Commentary"/>
+                                      
+                     <msItem xml:id="ms_i1.1">
+                              <locus from="2ra" to="37vb"/>
+                              <title type="complete" ref="LIT6608Commentary#Genesis"/>
+                              <textLang mainLang="gez"/>
+                              </msItem>
+
+
+                              <msItem xml:id="ms_i1.2">
+                                <locus from="38ra" to="54rb"/>
+                                 <title type="complete" ref="LIT6608Commentary#Exodus"/>
+                                 <textLang mainLang="gez"/>
+                              </msItem>
+
+                                 <msItem xml:id="ms_i1.3">
+                                    <locus from="54rb" to="64vb"/>
+                                    <title type="complete" ref="LIT6608Commentary#Leviticus"/>
+                                    <textLang mainLang="gez"/>
+                                 </msItem>
+
+                              <msItem xml:id="ms_i1.4">
+                                 <locus from="64vb" to="75ra"/>
+                                 <title type="complete" ref="LIT6608Commentary#Numbers"/>
+                                 <textLang mainLang="gez"/>                                 
+                             </msItem>
+                     
+                     <msItem xml:id="ms_i1.5">
+                        <locus from="75ra" to="84ra"/>
+                        <title type="complete" ref="LIT6608Commentary#Deuteronomy"/>
+                        <textLang mainLang="gez"/>                                 
+                     </msItem>
+                     
+                     <msItem xml:id="ms_i1.6">
```

Comment:
should it be msItem 1.6 or 2? Joshua is not Moses (LIT6608Commentary is Pentateuch)


or - probably better - we can have 
msItem 1 = Tergwame Orit (without ID - unless we create it)
  msItem 1.1 LIT6608Commentary
    msItem 1.1.1 LIT6608Commentary#Genesis
    msItem 1.1.2 LIT6608Commentary#Exodus
    msItem 1.1.3 LIT6608Commentary#Leviticus
    msItem 1.1.4 LIT6608Commentary#Numbers
    msItem 1.1.5 LIT6608Commentary#Deuteronomy
  msItem 1.2 LIT6817CommentJoshua

Reply:
Yes, it is more correct to have it like this.